server/src/internalClusterTest/java/org/elasticsearch/search/rank/MockedRequestActionBasedRerankerIT.java

```diff
@@ -321,33 +333,38 @@ public MockRequestActionBasedRankBuilder(
             final int rankWindowSize,
             final String field,
             final String inferenceId,
-            final String inferenceText
+            final String inferenceText,
+            final float minScore
         ) {
             super(rankWindowSize);
             this.field = field;
             this.inferenceId = inferenceId;
             this.inferenceText = inferenceText;
+            this.minScore = minScore;
         }
 
         public MockRequestActionBasedRankBuilder(StreamInput in) throws IOException {
             super(in);
             this.field = in.readString();
             this.inferenceId = in.readString();
             this.inferenceText = in.readString();
+            this.minScore = in.readFloat();
         }
 
         @Override
         protected void doWriteTo(StreamOutput out) throws IOException {
             out.writeString(field);
             out.writeString(inferenceId);
             out.writeString(inferenceText);
+            out.writeFloat(minScore);
```

Comment:
I believe that usually, we'd have to conditionally write new fields depending on the `TransportVersion` but since this completely new functionality (not used even in serverless) we should be ok.